src/akushon/config/utils/config.cpp

```diff
@@ -25,6 +25,7 @@
 
 #include "akushon/action/model/action_name.hpp"
 #include "akushon/config/utils/config.hpp"
+#include "jitsuyo/config.hpp"
```

Comment:
Should use angled brackets:
```suggestion
#include <jitsuyo/config.hpp>
```